docs/dasharo-tools-suite/documentation.md

```diff
@@ -544,6 +544,37 @@ New value is 0x8b.
 SPI Configuration is locked down
 ```
 
+#### Update issues
+
+If you see the following pop-ups during first the boot after the update:
+
+![error-0x03](./images/error-0x03.png)
+![error-0x13](./images/error-0x13.png)
+
+You probably performed an update using a deprecated version of Dasharo Tools
+Suite and have not disabled BIOS lock. Do not worry, nothing bad has happened.
+If you backed up your old firmware, do the following steps:
+
+1. Reboot your device and turn off BIOS lock (you can find this option in
+[Dasharo Security
+Options](../dasharo-menu-docs/dasharo-system-features.md#dasharo-security-options)
+as `BIOS boot medium lock`).
+1. Boot the DTS you backed up your old firmware with and choose option 3, which
+will restore it.
+1. After restoring the firmware, perform another update, but make sure you are
+using the **latest Dasharo Tools suite** version (new DTS has tools to protect
+you from such issues).
```

Comment:
Maybe let's link to the place where that `latest DTS` can be found

Reply:
@DaniilKl I think here we could link to GH release pages https://github.com/Dasharo/meta-dts/releases

Reply:
Added 6f4ac86d81f1b2c6a77368121b8e3d34b7496e82.